### DISCUSSION TITLE
iconfont不显示，Giscus一直Loading加载，但是本地编译预览没问题

### DISCUSSION THREAD

**Discussion by AniBullet** [vuepress-theme-hope/vuepress-theme-hope#2329] [Q&A] [answered]
https://animatorbullet.github.io/

iconAssets 试过 iconfont 或 css 链接，
Giscus 按文档设置了 app 和 仓库id，
换手机试了下也是一样症状，应该不是网络问题。
业余人士技术有限，求助~或者给个方向我自己去搜索排查也行，谢谢！

在线网站
![Snipaste_2022-10-23_14-37-46](https://user-images.githubusercontent.com/45991057/197377992-ad82e1b9-c768-48e0-885e-021063701886.png)

本地 localhost 预览
![Snipaste_2022-10-23_14-37-32](https://user-images.githubusercontent.com/45991057/197377995-93ec2833-3153-4095-8250-c2f7f8733d1d.png)

**Comment by Mister-Hope** [comment, marked as answer]
See https://github.com/vuepress-theme-hope/vuepress-theme-hope/issues/2327, revert to vp2@beta51 and our beta.110

**Reply by AniBullet** [reply]
谢谢~

**Reply by AniBullet** [reply]
```
  "devDependencies": {
    "@vuepress/client": "2.0.0-beta.51",
    "vue": "^3.2.41",
    "vuepress": "2.0.0-beta.51",
    "vuepress-plugin-comment2": "2.0.0-beta.110",
    "vuepress-theme-hope": "2.0.0-beta.110"
  }
```

刚才看了下我现在好像是 beta.51 和 beta.110，但是网页还是不正常

**Reply by AniBullet** [reply]
把文件全部移走，重新安装新的主题（文件夹层级和设置文件都有变动）再重新把文档和设置迁移回来（还好文档不多），就解决了，还是感谢一下~
